qiskit/pulse/builder.py

```diff
@@ -104,14 +104,12 @@
 
 Methods to return the correct channels for the respective qubit indices.
 
-.. plot::
-   :include-source:
-   :nofigs:
+.. code-block:: python
```

Comment:
I understand that you skipped the execution of the examples here because they won't work without calibrations. It is a bit strange to keep an example we know is incorrect, but I don't feel too strongly about it given that the class will be removed shortly anyways.

Reply:
Exactly. The main motivation to do it like this is to keep the changes here as minimal as possible w.r.t the scope of this PR